pallets/ajuna-awesome-avatars/src/impls/treasury_manager.rs

```diff
@@ -35,12 +35,4 @@ impl<T: Config> TreasuryManager for Pallet<T> {
 	fn get_treasurer_for(_key: Self::TreasuryPotKey) -> Result<Self::AccountId, DispatchError> {
 		Ok(Self::treasury_account_id())
 	}
-
-	fn deposit_into(
-		depositor: &Self::AccountId,
-		key: &Self::TreasuryPotKey,
-		fee: Self::Currency,
-	) -> Result<(), DispatchError> {
-		Self::deposit_into_treasury(depositor, key, fee)
-	}
```

Comment:
I decided to currently remove the usage of the treasury, as it is not clear to me, how we are gonna actually implement yet. I want to tackle the design of it this week.